terraform_readme.sh

```diff
@@ -0,0 +1,47 @@
+#!/usr/bin/env bash
+set -e
+
+declare -a paths
+declare -a tfvars_files
+
+index=0
+
+for file_with_path in "$@"; do
+  file_with_path="${file_with_path// /__REPLACED__SPACE__}"
+
+  paths[index]=$(dirname "$file_with_path")
+
+  if [[ "$file_with_path" == *".tfvars" ]]; then
+    tfvars_files+=("$file_with_path")
+  fi
+
+  let "index+=1"
+done
+
+for path_uniq in $(echo "${paths[*]}" | tr ' ' '\n' | sort -u); do
+  path_uniq="${path_uniq//__REPLACED__SPACE__/ }"
+
+  pushd "$path_uniq" > /dev/null
+  if [[ -f README.md ]]; then
+    if grep -q "BEGINNING OF TERRAFORM-DOCS HOOK" README.md; then
+      DOCS=$(terraform-config-inspect ./)
```

Comment:
change between this and terraform docs is replacing terraform-docs md with terraform-config-inspect